### PR TITLE
adding submit & help link to tutorial

### PR DESCRIPTION
**Pull request by SidelineCory24**
Tutorial has been updated.

To-do:
- [x] recording a readable gif
- [x] rerecord gifs

### CODE REVIEW

File: src/Components/ResultsPage/tutorialSteps.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
+import { getAssetPath } from '../../utilities';
```
Comment (mjoyce91) [inline src/Components/ResultsPage/tutorialSteps.js:1]: this can just be `from 'utilities';`

Comment (SidelineCory24) [reply src/Components/ResultsPage/tutorialSteps.js:1]: done

**Comment by SidelineCory24** [comment]
Gifs are more zoomed in and should include the cursor being shown now

File: src/Components/Footer/Footer.jsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ const Footer = () => (
                 <Link className="usa-footer-primary-link" to="/">Home</Link>
               </li>
               <li className="usa-width-one-sixth usa-footer-primary-content">
-                <Link className="usa-footer-primary-link" to="/help">Help</Link>
+                <Link className="usa-footer-primary-link tutorial-help-link" to="/help">Help</Link>
```
Comment (elizabeth-jimenez) [inline src/Components/Footer/Footer.jsx:22]: 🎊

**Comment by elizabeth-jimenez** [comment]
image
@mjoyce91 @SidelineCory24 do y'all think we should change "skip" to "exit"?

File: src/Components/ResultsPage/tutorialSteps.js [hunk 3]
```diff
@@ -1,3 +1,9 @@
+import { getAssetPath } from 'utilities';
+
+const viewPositionText = 'To view more information about a position and add it to your bid list, click "View position". To submit a bid, you will need go to your Bid Tracker.';
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsPage/tutorialSteps.js:3]: maybe we should say 'To view more information about a position and add it to your bid list, click "View position". ~To submit a bid, you will need go to your Bid Tracker~ See next step for how to submit a bid after adding it to your Bid Tracker.'?

File: src/Components/ResultsPage/tutorialSteps.js [hunk 4]
```diff
@@ -34,7 +40,25 @@ const steps = [
...
+    content:
+      <div>
+        <div>Submitting a bid on the Bid Tracker</div>
+        <img alt="submit bid gif" src={submitBidGif} />
+      </div>,
+    hideCloseButton: true,
+  },
+  {
+    target: '.tutorial-help-link',
+    content: 'Click the "Help" link to email the TalentMAP team any questions.',
```
Comment (elizabeth-jimenez) [inline src/Components/ResultsPage/tutorialSteps.js:61]: 'Click the "Help" link for FAQs and additional support ~to email the TalentMAP team any questions~.'?

Comment (elizabeth-jimenez) [reply src/Components/ResultsPage/tutorialSteps.js:61]: i would make sure they do have FAQs on the prd Help page, i think they do but 🤔

File: src/Components/ResultsPage/tutorialSteps.js (hunk 3 above)
Comment (SidelineCory24) [reply src/Components/ResultsPage/tutorialSteps.js:3]: done

**Comment by elizabeth-jimenez** [comment]
image
on this gif, would it be possible to start at the zoomed out version of the bid tracker and then zoom in a bit to click the button? I feel like that'll make it a lot more recognizable "oh i know where that is". Bonus points if the menu shows!
I really like how you pieced in the state of the bid tracker after a successful Submit 🎉

File: src/Components/ResultsPage/tutorialSteps.js (hunk 4 above)
Comment (SidelineCory24) [reply src/Components/ResultsPage/tutorialSteps.js:61]: done. I did not know what to say here at all lol

Comment (SidelineCory24) [reply src/Components/ResultsPage/tutorialSteps.js:61]: @mjoyce91 are there FAQs in prod?

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez gif has been updated and text has been changed from skip -> exit.

**Review by elizabeth-jimenez — APPROVED** [review]

**Comment by elizabeth-jimenez** [comment]
Do Not Merge pending feedback from Mike:
FAQs
Skip -> Exit
Blue cursor on 2nd gif inconsistent with first, but up to y'all!

**Comment by mjoyce91** [comment]
@elizabeth-jimenez @SidelineCory24 

Yes there are FAQs in PRD, so that wording is good
Skip/Exit wasn't directly related to this PR, so let's do that separate
Let's make the gifs consistent with the cursor - I'd recommend excluding the bubble

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez @SidelineCory24
> 
> Yes there are FAQs in PRD, so that wording is good Skip/Exit wasn't directly related to this PR, so let's do that separate Let's make the gifs consistent with the cursor - I'd recommend excluding the bubble

"Skip/Exit wasn't directly related to this PR, so let's do that separate" was already done, should we revert?

**Comment by mjoyce91** [comment]
@elizabeth-jimenez - oh if it's already done then that's fine

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez @mjoyce91 "Exit" over "Skip" is done and bubble has been removed

**Pull request merged**